src/test/java/org/imsglobal/caliper/ri/events/GradeEventGradedTest.java

```diff
@@ -0,0 +1,155 @@
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+
+import static com.yammer.dropwizard.testing.JsonHelpers.jsonFixture;
+import static org.imsglobal.caliper.ri.events.HMHConstants.ACTIVITY_REF_ID;
+import static org.imsglobal.caliper.ri.events.HMHConstants.APP_NAME;
+import static org.imsglobal.caliper.ri.events.HMHConstants.BASE_IRI;
+import static org.imsglobal.caliper.ri.events.HMHConstants.BASE_URN;
+import static org.imsglobal.caliper.ri.events.HMHConstants.DISTRICT_REF_ID;
+import static org.imsglobal.caliper.ri.events.HMHConstants.OBJECT_ID;
+import static org.imsglobal.caliper.ri.events.HMHConstants.SCHOOL_REF_ID;
+import static org.imsglobal.caliper.ri.events.HMHConstants.STUDENT_NAME;
+import static org.imsglobal.caliper.ri.events.HMHConstants.STUDENT_USER_REF_ID;
+
+@Category(org.imsglobal.caliper.UnitTest.class)
+public class GradeEventGradedTest {
+    private JsonldContext context;
+    private String id;
+    private SoftwareApplication edApp;
+    private Person actor;
+    private Attempt object;
+    private Assessment assignable;
+    private Score generated;
+    private Membership membership;
+    private GradeEvent event;
+
+    @Before
+    public void setUp() throws Exception {
+        context = JsonldStringContext.getDefault();
+
+        id = "urn:uuid:a50ca17f-5971-47bb-8fca-4e6e6879001d";
+
+        //actor = SoftwareApplication.builder().id(BASE_IRI.concat(APP_NAME)).version("v2").build();
+        actor = Person.builder().id(BASE_URN.concat(STUDENT_USER_REF_ID)).name(STUDENT_NAME).build();
+        assignable = Assessment.builder().id(BASE_URN.concat(ACTIVITY_REF_ID)).build();
+
+
+
+        object = Attempt.builder()
+            .id(BASE_URN.concat(OBJECT_ID))
+            .assignable(assignable)
+            .assignee(Person.builder().id(actor.getId()).coercedToId(true).build())
+            .count(35)
+            .dateCreated(new DateTime(2016, 11, 15, 10, 5, 0, 0, DateTimeZone.UTC))
+            .startedAtTime(new DateTime(2016, 11, 15, 10, 5, 0, 0, DateTimeZone.UTC))
+            .endedAtTime(new DateTime(2016, 11, 15, 10, 55, 12, 0, DateTimeZone.UTC))
+            .build();
+
+        generated = Score.builder()
+            .id(BASE_URN.concat(ACTIVITY_REF_ID))
+            .attempt(Attempt.builder().id(object.getId()).coercedToId(true).build())
+            .maxScore(1700)
+            .scoreGiven(800)
+            .scoredBy(SoftwareApplication.builder().id(BASE_IRI.concat(APP_NAME)).coercedToId(true).build())
+            .comment("The lexile score from Test Quiz # "+OBJECT_ID+" taken on date/time for student " + STUDENT_USER_REF_ID)
+            .dateCreated(new DateTime(2016, 11, 15, 10, 56, 0, 0, DateTimeZone.UTC))
+            .build();
+
+        edApp = SoftwareApplication.builder().id(BASE_IRI.concat(APP_NAME)).coercedToId(true).build();
+
+        membership = Membership.builder()
+                .id(actor.getId())
```

Comment:
the id here of the membership, isn't the actor id - im not sure how it can represented but its not actor id anyway